### ISSUE TITLE
Sonoff Mini DIY and Alexa...

### ISSUE THREAD

**Issue by d-sudhakar** [itead/Sonoff_Devices_DIY_Tools#39]
Does the Alexa skills work with Sonoff Mini when it is in the DIY mode (jumper inserted)? I tried discovering it using Alexa app but I think it looks for eWeLink support which is disabled in DIY mode. Please advice if anyone got Alexa worked with Mini in DIY mode.

**Comment by joshhodgson** [comment]
Not supported in the default firmware. Other firmware is available such as Tasmota

**Comment by d-sudhakar** [comment]
Yeah, 3rd party s/w such as Tasmota, Espurna do provide support for Alexa, but the device will be out of warranty soon after flashing the 3rd party s/w. I was checking if there is any plan for providing Alexa support in Sonoff DIY firmware.

**Comment by joshhodgson** [comment]
You could write an application using FauxMo to run on a raspberry pi to
relay messages to the sonoff

**Comment by d-sudhakar** [comment]
Agree that Alexa -> raspberry pi -> sonoff solution works. 
Still it would be nice to have a direct link between Alexa and sonoff rather than introducing an intermediary.

**Comment by joshhodgson** [comment]
The standard sonoff firmware does support Alexa. What you're asking for is
a DIY firmware where things are done for you 😛 What would be nice is a DIY
mode as part of the original firmware where you can do both

**Comment by mattsaxon** [comment]
The standard firmware does still support the REST API. That way you can have local control and still use eWeLink.
